examples/fscloud/variables.tf

```diff
@@ -69,3 +69,9 @@ variable "allow_at_to_cos" {
   description = "Set rule for Activity Tracker to COS, default is true"
   default     = true
 }
+
+variable "allow_iks_to_is" {
+  type        = bool
+  description = "Set rule for IKS to IS, default is true"
```

Comment:
I think IS should be explain here as not fully obvious to someone not familiar with the underlaying CBR naming terminology.

Reply:
Done